### PR TITLE
feat: dashboard hints

### LINKED ISSUE #180: Dashboard page hints
Display hints on dashboard page and convert existing ones to present tense.

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves #180 
Implements remaining dashboard hints except transactions table (as agreed with @michele-franchi)

## Demo
image
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

File: src/utils/hints/namesHints.js [hunk 1]
```diff
@@ -23,4 +23,8 @@ export const namesHints = {
   eventTransactionHash: 'Transaction hash of the Name event.',
   eventTime: 'Block height and estimated date and time when the event was triggered.',
   activity: 'Type of event that was triggered.',
+  recentlyActivatedData: 'The Name and its owner who recently activated it. Depending on the length a Name is activated either through an auction or by direct acquisition.',
```
Comment (janmichek) [inline src/utils/hints/namesHints.js:26]: Here I would explain precisely how it works (13 characters threshold). Personally I would use some simpler word than acquisition

File: src/utils/hints/namesHints.js [hunk 2]
```diff
@@ -23,4 +23,8 @@ export const namesHints = {
...
+  activationPrice: 'Amount of AE tokens than have been burned during Name activation.',
```
Comment (janmichek) [inline src/utils/hints/namesHints.js:27]:
```suggestion
  activationPrice: 'Amount of AE tokens paid for the Name activation.',
```

Is it really 'burned' ?

**Review by janmichek — CHANGES_REQUESTED** [review]
Good job, ser! Please consider my suggestions.

One more thing from the previous PR

In state channels hints, there should be also explained responder in:

`participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',`
Could you improve it as well?

**Comment by janmichek** [comment]
How about hints in here? Should it be part of this PR?
image

File: src/utils/hints/namesHints.js (hunk 2 above)
Comment (lukeromanowicz) [reply src/utils/hints/namesHints.js:27]: Isn't it burned? :O

File: src/utils/hints/stateChannelsHints.js [hunk 3]
```diff
@@ -6,4 +6,6 @@ export const stateChannelsHints = {
   locked: 'Currently deposited amount to the State Channel by its participants.',
   lastUpdated: 'Block height and estimated date and time of the last interaction with the State Channel.',
   lastTxType: 'Last transaction\'s type of the State Channel.',
+  initiator: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
```
[the remark this answers is not in the document]
Comment (lukeromanowicz) [reply src/utils/hints/stateChannelsHints.js:9]: I'll update it to use the version we agreed upon in state channel details hints PR

**Comment by lukeromanowicz** [comment]
@janmichek thanks for your suggestion. The transactions section is skipped until they are implemented in another task. If by the time this task is mergable they will be still in development - I'll create an appropriate task to implement them. Otherwise I will add them before merging

**Review by janmichek — APPROVED** [review]
Cool, thanks for fixes ✌️

**Review by CharisSiarampalis — APPROVED** [review]

**Review by michele-franchi — APPROVED** [review]
Looks good!

**Pull request merged**